src/librustdoc/html/format.rs

```diff
@@ -1160,7 +1159,7 @@ impl clean::Import {
     crate fn print(&self) -> impl fmt::Display + '_ {
         display_fn(move |f| match self.kind {
             clean::ImportKind::Simple(ref name) => {
-                if *name == self.source.path.last_name() {
+                if name.as_str() == self.source.path.last_name() {
```

Comment:
Realizing that now it should be instead: `if name == self.source.path.last() {`. Updating once confirmed.

Reply:
What's the difference between the two?

Reply:
In the original code I was comparing strings whereas in the second we compare `Symbol` (so usize). I assume the second one to be faster. ;)